### PR TITLE
Listing page

### PR DESCRIPTION
**Pull request by jacquescore**
Added listing page when you click on a search result

### CODE REVIEW

File: src/routes/listings/listingsView.js [hunk 1]
```diff
@@ -20,11 +22,47 @@ router.get("/search", authSafe, async (req, res, next) => {
 
   try {
     const result = await getAll(query);
+    console.log(result)
```
Comment (nuendo2004) [inline src/routes/listings/listingsView.js]: remove console.log

File: src/routes/listings/listingsView.js [hunk 2]
```diff
@@ -20,11 +22,47 @@ router.get("/search", authSafe, async (req, res, next) => {
...
     if (!result) throw new NotFoundException(`listing not found`);
+    console.log(result._id)
```
Comment (nuendo2004) [inline src/routes/listings/listingsView.js]: remove

File: src/routes/listings/listings.js [hunk 3]
```diff
@@ -14,8 +16,8 @@ router.post("/create", auth, async (req, res, next) => {
   try {
     const validListing = Validator.validateListing(req.body);
 
-    if (!req.user.role.includes("seller"))
-      throw new ValidationException("User is not a seller");
+    //if (!req.user.role.includes("seller"))
```
Comment (nuendo2004) [inline src/routes/listings/listings.js]: what was the reason comment it out?

File: src/views/home.handlebars
Comment (nuendo2004) [inline src/views/home.handlebars:1]: No need for boilerplate

File: src/views/listingDetails.handlebars
Comment (nuendo2004) [inline src/views/listingDetails.handlebars:1]: No need for boilerplate, also no body tag as the whole view is actually already in a body tag

**Review by nuendo2004 — CHANGES_REQUESTED** [review]

File: src/routes/listings/listings.js (hunk 3 above)
Comment (jacquescore) [reply src/routes/listings/listings.js]: Oh sorry, i forgot it said: Property 'role' does not exist on type '{ _id: string; firstName: string; lastName: string; userName: string; password: string; avatar: string; email: string; address: string; }'.ts(2339)

**Review by nuendo2004 — APPROVED** [review]

File: src/routes/listings/listings.js (hunk 3 above)
Comment (nuendo2004) [reply src/routes/listings/listings.js]: I should be there, check the user schema and database, make sure it has role field

**Pull request merged**